docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -14,63 +14,58 @@ endif::[]
 :icons: font
 :toc: macro
 
+
 Before devices can receive updates, each device needs to have a unique identity and a certificate. The provisioning process ensures that a signed certificate is associated with each device. This process is crucial for securing communication between devices and the OTA Connect server.
-OTA Connect supports two provisioning methods. These methods determine the components that will play the role of the "issuer" and the "verifier" in your infrastructure. The "issuer" is the server that signs and issues your device certificates. The "verifier" is the server that verifies the authenticity of device certificates.
-We refer these two methods as "provisioning with shared credentials" and "provisioning with device credentials".
 
-* *Provisioning with shared credentials*
-+
-This type of provisioning is great for testing because the OTA Connect server plays the role of both the "issuer" and "verifier" when creating device certificates. You can try out OTA Connect without involving the rest of your infrastructure. However, it's not secure enough for a production scenario.
-+
-* *Provisioning with device credentials*
-+
-This is the provisioning method that you should use in production. In this scenario, the OTA Connect server doesn't issue any credentials to devices; it simply inspects device certificates that come preinstalled. The OTA Connect server only plays the role of "verifier" and the "issuer" role is handled by your fleet certificate authority.
+OTA Connect supports two provisioning methods. These methods determine the components that will issue and verify the certificate in your infrastructure. We refer to these two methods as "*provisioning with shared credentials*" and "*provisioning with device credentials*". The end result of these two provisioning methods is the same, as they both provide a device with a signed certificate that can be used for communication.
+
+
+For secure communication, a vehicle needs to have an X.509 certificate and it needs to trust the device gateway’s X.509 certificate. (Each account has its own unique device gateway, and that certificate is included in the credentials.zip and baked into the image as a pinned CA when you bitbake.) The device gateway trusts a vehicle’s X.509 certificate if and only if that certificate is signed by what we call the fleet root certificate authority (CA). Every device, regardless of how it is initially provisioned, ends up with its own unique X.509 certificate signed by the fleet root CA. Whenever the device interacts with the OTA Connect server, the device gateway authenticates and identifies it by its X.509 certificate.
```

Comment:
> Each account has its own unique device gateway

My understanding was that each account has its own unique *cert* or *endpoint* on the device gateway, but there is only one instance of the service.

Also, "CA" is used before the acronym is written out in full, although it's in the parenthetic section. Maybe you don't actually need the parentheses?

Reply:
I will amend this section slightly to mention each device having a unique cert on the device gateway. However should I also state that there is only one instance of the service? Or is it not important for the customer to be aware of this?

Reply:
> However should I also state that there is only one instance of the service?

I don't think it needs to be stated explicitly; I'm not sure the user would care, and it's probably explained better somewhere else. Just referring to the DevGW in the singular is fine, I think.

Reply:
ok. no problem.

Reply:
> My understanding was that each account has its own unique _cert_ or _endpoint_ on the device gateway, but there is only one instance of the service.

I think it might have been me who put this language initially. While it's true that there's only one instance of the service, that's an interior view that the customer doesn't really need to know about. Every account has its own unique device gateway URL which nobody else's devices connects to; in my view, that's enough to describe it as "your" device gateway in this high-level view.